docs/src/SubNav.js

```diff
@@ -53,9 +53,16 @@ class SubNav extends React.Component {
     }
 
     if (props.children) {
-      return ValidComponentChildren.some(props.children, child => (
-        this.isActive(child, activeKey, activeHref)
-      ));
+      let active;
+      ValidComponentChildren.forEach(props.children, (child) => {
+        if (active === undefined) {
+          active = this.isActive(child, activeKey, activeHref);
+        }
+      });
+
+      if (active !== undefined) {
```

Comment:
if you already have this check maybe use `some` above? that or just add a note to the nuance of this. it wasn't obvious to me why this was needed, re: undefined vs false